pkg/frontend/transport/handler.go

```diff
@@ -122,6 +124,16 @@ func NewHandler(cfg HandlerConfig, roundTripper http.RoundTripper, log log.Logge
 			Help: "Number of samples fetched to execute a query.",
 		}, []string{"user"})
 
+		h.queryTotalQueryableSamples = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_query_total_queryable_samples_total",
+			Help: "Number of total queryable samples to execute a query.",
```

Comment:
I feel queryable samples is a bit confusing.
What about `cortex_query_samples_processed_total` or `cortex_query_samples_scanned_total`? 
You can add a comment to reference it to https://github.com/prometheus/prometheus/blob/main/util/stats/query_stats.go#L237

Reply:
For me, `cortex_query_samples_processed_total` is more good. Thank you!

Reply:
How about changing the variable name `queryTotalQueryableSamples` to `queryProcessedSamples` and `querySamples` to `queryFetchedSamples`?

Reply:
I think we can keep `queryTotalQueryableSamples` as it is clear that it refers to `Total Samples`.
Makes sense to rename `querySamples` to `queryFetchedSamples`.